backend/stewards/migrations/0010_builder_project_gate_templates.py

```
@@ -0,0 +1,85 @@
+    ReviewTemplate = apps.get_model('stewards', 'ReviewTemplate')
+    for template in TEMPLATES:
+        ReviewTemplate.objects.update_or_create(
+            label=template['label'],
+            defaults={
+                'text': template['text'],
+                'action': template['action'],
+            },
+        )
+
+
+def remove_templates(apps, schema_editor):
```

Ruff (0.15.15)

[warning] Unused function argument: `schema_editor`

(ARG001)